### DISCUSSION TITLE
How to put TMD files into the .fst file for calculation and operation.

### DISCUSSION THREAD

**Discussion by zdraemrain** [OpenFAST/openfast#1299] [Q&A] [answered]
How to put TMD files into the .fst file for calculation and operation.

**Comment by jjonkman** [comment, marked as answer]
Dear @zdraemrain,

You must specify how many TMDs you want to include and where (blade, nacelle, tower, substructure) in the ServoDyn input file as documented here: https://openfast.readthedocs.io/en/main/source/user/servodyn/input.html#servodyn-primary-input-file.  And then for each TMD, you specify the configuration details in the Structural Control (StC) input file as documented here: https://openfast.readthedocs.io/en/main/source/user/servodyn-stc/StC_index.html.

Best regards,

**Reply by zdraemrain** [reply]
Dear @jjonkman ,

Thank you for your reply, I'll give it a try.

Best wishes!
